develop/parachains/maintenance/unlock-parachain.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Unlock a Parachain
+description: Learn how to unlock your parachain. This step-by-step guide covers verifying lock status, preparing calls, and executing the unlock process.
+---
+
+# Unlock a Parachain
+
+## Introduction
+
+Parachain locks are a critical security mechanism in the Polkadot ecosystem designed to maintain decentralization during the parachain lifecycle. These locks prevent potential centralization risks that could emerge during the early stages of parachain operation.
+The locking system follows strict, well-defined conditions that distribute control across multiple authorities:
+
+The locking system follows strict, well-defined conditions that distribute control across multiple authorities:
+
+- Relay chain governance has the authority to lock any parachain
+- A parachain can lock its own lock
+- Parachain managers have permission to lock the parachain
+- Parachains are locked automatically when they successfully produce their first block
+
+Similarly, unlocking a parachain follows controlled procedures:
+
+- Relay chain governance retains the authority to unlock any parachain
+- A parachain has the ability to unlock its own lock
+
+This document guides you through the process of checking a parachain's lock status and safely executing the unlock procedure from a parachain using XCM (Cross-Consensus Messaging).
+
+## Check if the Parachain is Locked
+
+Before attempting to unlock a parachain, you should verify its current lock status. This can be done through the Polkadot.js interface:
+
+1. In [Polkadot.js Apps](https://polkadot.js.org/apps/#/explorer){target=\_blank}, connect to the relay chain, navigate to the **Developer** dropdown and select the **Chain State** option
+
+2. Query the parachain locked status:
+    1. Select **`registrar`**
+    2. Choose the **`paras`** option
+    3. Input the parachain ID you want to check as a parameter (e.g. `2006`)
+    4. Click the **+** button to execute the query
+    5. Check the status of the parachain lock
+
+    ![](/images/develop/parachains/maintenance/unlock-parachain/unlock-parachain-1.webp)
+
+## How to Unlock a Parachain
+
+Unlocking a parachain requires sending an XCM (Cross-Consensus Message) to the relay chain from the parachain itself, sending a message with Root origin, or this can be accomplished through the relay chain's governance mechanism executing a root call.
+
+If sending an XCM, the parachain origin must have proper authorization, which typically comes from either the parachain's sudo pallet (if enabled) or through its governance system.
```

Comment:
```suggestion
Unlocking a parachain requires sending an XCM (Cross-Consensus Message) to the relay chain from the parachain itself, sending a message with Root origin, or this can be accomplished through the relay chain's governance mechanism, executing a root call.

If sending an XCM, the parachain origin must have proper authorization, typically from either the parachain's sudo pallet (if enabled) or its governance system.
```